packages/vapor/scss/components/logo-card.scss

```diff
@@ -29,15 +28,18 @@
 .logo-card-content {
     display: flex;
     flex-direction: column;
-    justify-content: space-between;
-}
-
-.logo-card-content-center {
-    display: flex;
-    flex-direction: column;
+    flex-grow: 1;
+    align-items: left;
```

Comment:
This seems to be an invalid property value
![image](https://user-images.githubusercontent.com/35579930/105412140-4f984000-5c02-11eb-84bf-02318523d324.png)